docs/creating.rst

```diff
@@ -23,3 +23,11 @@ Any validating function that validates against a subschema should call
 instance, or schema, it should pass one or both of the ``path`` or
 ``schema_path`` arguments to ``descend`` in order to properly maintain
 where in the instance or schema respectively the error occurred.
+
+The Validator Protocol
+----------------------
+
+``jsonschema`` defines a protocol, ``jsonschema.protocols.IValidator`` which
```

Comment:
I think we can drop the `I` prefix -- it was here because in the `zope.interface` world the convention is to use this Hungarian-ish prefix for interfaces, but I don't think with PEP544 protocols that people tend to use it they just probably call the thing what it is with no prefix (unless you know of other cases).

Reply:
Okay, that's good to know. I actually thought it was short for "InstanceValidator"!

I didn't want to jump to rename without discussing it, but am more than happy to do so.